### DISCUSSION TITLE
AWS DocumentDB support

### DISCUSSION THREAD

**Discussion by agoldis** [sorry-cypress/sorry-cypress#327] [General]
@pbeckham, a follow up from #324 

There's a chance future changes would not be 100% compatible with DocumentDB. "officially" supporting two implementations would create more maintenance burden - I am trying to avoid that :)

Do you have any idea - how can we automate integration with DocumentDB?

**Comment by pbeckham** [comment]
@agoldis thank you for reviewing and merging the PR so quickly!

Yeah I am unsure how compatibility with DocumentDB could be checked automatically. I have sent a question to the AWS DocumentDB team, and will let you know here if I hear back from them.

I agree automatic tooling would be the best way to go to support the two implementations. Will continue looking around, but not finding an obvious solution here.
